test/wanda_web/controllers/v2/execution_controller_test.exs

```diff
@@ -11,6 +11,173 @@ defmodule WandaWeb.V2.ExecutionControllerTest do
 
   setup :verify_on_exit!
 
+  describe "list executions" do
```

Comment:
These tests are copied from the `v1` version controller tests. At the end, they need to test the same thing.
I don't have right now any option to have some sort of generic code to test the same in both modules.
Any ideas?

Reply:
Top of my mind, I think that having the same test would be fine, but I have no strong opinion about that :+1: